app/src/main/java/com/asfoundation/wallet/ui/iab/PaymentMethodsFragment.kt

```diff
@@ -182,9 +182,22 @@ class PaymentMethodsFragment : DaggerFragment(), PaymentMethodsView {
     setupSubject!!.onNext(true)
   }
 
+  override fun onResume() {
+    if (paymentMethodList.isNotEmpty()) {
+      showLoading()
+    }
+    presenter.setupUi(transactionValue.toDouble())
```

Comment:
Altough this isn't wrong in the classic MVP archictecture, I think we should avoid for the fragment to know presenters methods as much as possible to be "loyal" to our MVP architecture. So in this case I would have a presenter method called onResume that would have the setupUi in there